cmd/kubeconfigcmd/kubeconfig.go

```diff
@@ -71,7 +74,8 @@ gcloud auth login --update-adc`,
 			return kubeconfig.CreateKubeconfig(context.Context, email, tenant,
 				kubeconfig.WithOverwriteData(overwrite),
 				kubeconfig.WithFromScratch(clear),
-				kubeconfig.WithOnpremClusters(includeOnprem),
+				kubeconfig.WithExcludeClusters(exclude),
+				kubeconfig.WithOnpremClusters(true),
```

Comment:
Kunne vurdert å fjerne `WithOnpremClusters` helt.

Reply:
Tenkte å lage en annen PR, for så det var litt rariteter her: https://github.com/nais/cli/blob/12c9776d94dbc9ac6b78c1090c9cfd1961c05ca5/pkg/kubeconfig/filter.go#L23-L45 
Alle optionsa setter samme bool på config structen :)

Reply:
Støttes. Er en del opprydding rundt andre ting som er veldig nav-spesifikke og